### PR TITLE
Bug fix for buckets card and renamed function for readability

### PR DESCRIPTION
**Pull request by afreen23**
- corrected the unhealthy bucket claims formula
 - made link open in new tab
 - renamed getPropsData function to getGaugeValue

### CODE REVIEW

**Comment by afreen23** [comment]
@cloudbehl @gnehapk please review

File: frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status,
       <span className="co-dashboard-text--small nb-buckets-card__row-subtitle">Unavailable</span>
     ) : Number(status) > 0 ? (
       <React.Fragment>
-        <a href={link} style={{ textDecoration: 'none' }}>
+        <a href={link} style={{ textDecoration: 'none' }} target="blank">
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx]: Make this a external link. make sure all the noobaa dashboard links are external.

File: frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx [hunk 2]
```diff
@@ -65,23 +79,32 @@ const ObjectDashboardBucketsCard: React.FC<DashboardItemProps> = ({
...
   ]);
 
+  const obcData = _.get(resources.obc, 'data', null) as K8sResourceKind[];
   const noobaaSystemAddress = getMetric(bucketsLinksResponse, 'system_address');
   const noobaaSystemName = getMetric(bucketsLinksResponse, 'system_name');
+  const obcCount = getGaugeValue(obcCountResponse);
+  const unhealthyObcCount = getGaugeValue(unhealthyObcCountResponse);
+
+  let resultantUnhealthyObcCount: number = null;
   let link = null;
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx]: type declaration

File: frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx (hunk 1 above)
Comment (afreen23) [reply frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx]: its external !

Comment (cloudbehl) [reply frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx]: https://github.com/openshift/console/blob/master/frontend/public/components/nav/items.tsx#L22..L24

Comment (afreen23) [reply frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx]: how would I wrap this in`ExternalLink`
```
<RedExclamationCircleIcon className="co-dashboard-icon nb-bucket-card__status-icon" />
 <span className="nb-buckets-card__row-status-item-text">{status}</span>
```

**Pull request merged**